packages/compass-components/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "allowJs": false,
+    "esModuleInterop": true,
+    "downlevelIteration": true,
+    "declaration": true,
+    "jsx": "react",
+    "lib": [
+      "dom",
+      "es2018"
+    ],
+    "module": "commonjs",
+    "outDir": "lib",
+    "removeComments": true,
+    "strictNullChecks": true,
+    "sourceMap": true,
+    "strict": true,
```

Comment:
Having `strict` and `strictNullChecks` set to true is a bit redundant and can be omitted. Strict option [activates all `strict*` options](https://www.typescriptlang.org/tsconfig#strict)